src/components/installer/installationlayout.ts

```diff
@@ -17,9 +17,21 @@ export function formatBin(tool: string, platform: Platform): string | null {
     if (!platformString) {
         return null;
     }
-    const toolPath = `${platformString}-amd64/${tool}`;
+    const platformArchString = platformArch(platformString);
+    const toolPath = `${platformString}-${platformArchString}/${tool}`;
     if (platform === Platform.Windows) {
         return toolPath + '.exe';
     }
     return toolPath;
 }
+
+export function platformArch(os: string) {
+    if (os !== 'linux') {
```

Comment:
This might be a silly question, but is there any reason to only check the architecture on Linux?  Why not just check regardless of the OS?

Reply:
I saw that there are tools that are only packaged for amd64. Kubectl only comes in amd64 for Darwin and Windows.